2022-03-03 00:04:13.274 ERROR 118 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:58254
2022-03-03 14:43:49.740  INFO 119 --- [main] lavalink.server.Launcher                 : 

[32m       .  [31m _                  _ _       _    [32m__ _ _
[32m      /\\ [31m| | __ ___   ____ _| (_)_ __ | | __[32m\ \ \ \
[32m     ( ( )[31m| |/ _` \ \ / / _` | | | '_ \| |/ /[32m \ \ \ \
[32m      \\/ [31m| | (_| |\ V / (_| | | | | | |   < [32m  ) ) ) )
[32m       '  [31m|_|\__,_| \_/ \__,_|_|_|_| |_|_|\_\[32m / / / /
[0m    =========================================[32m/_/_/_/[0m

	Version:        5f76fd37dd29f9ada64d8b78475a367e5e52b71d-SNAPSHOT
	Build:          Unofficial
	Build time:     05.11.2021 22:07:41 UTC
	Branch          dev
	Commit:         5f76fd3
	Commit time:    05.11.2021 22:07:23 UTC
	JVM:            16
	Lavaplayer      1.3.96-original

2022-03-03 14:43:49.926  INFO 119 --- [main] lavalink.server.Launcher                 : Starting Launcher using Java 16 on fdd1318f24af with PID 119 (/home/runner/LavaLink-with-Replit/Lavalink.jar started by runner in /home/runner/LavaLink-with-Replit)
2022-03-03 14:43:49.927  INFO 119 --- [main] lavalink.server.Launcher                 : No active profile set, falling back to default profiles: default
2022-03-03 14:43:50.201  WARN 119 --- [background-preinit] o.s.h.c.j.Jackson2ObjectMapperBuilder    : For Jackson Kotlin classes support please add "com.fasterxml.jackson.module:jackson-module-kotlin" to the classpath
2022-03-03 14:43:52.389  WARN 119 --- [main] io.undertow.websockets.jsr               : UT026010: Buffer pool was not set on WebSocketDeploymentInfo, the default pool will be used
2022-03-03 14:43:52.433  INFO 119 --- [main] io.undertow.servlet                      : Initializing Spring embedded WebApplicationContext
2022-03-03 14:43:52.433  INFO 119 --- [main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 2338 ms
2022-03-03 14:43:52.644  INFO 119 --- [main] l.server.metrics.PrometheusMetrics       : Prometheus metrics set up
2022-03-03 14:43:52.964  INFO 119 --- [main] c.s.d.l.tools.GarbageCollectionMonitor   : GC monitoring enabled, reporting results every 2 minutes.
2022-03-03 14:43:53.050  WARN 119 --- [main] l.server.config.SentryConfiguration      : Turning off sentry
2022-03-03 14:43:53.393  INFO 119 --- [main] lavalink.server.config.KoeConfiguration  : OS: Linux, Arch: amd64
2022-03-03 14:43:53.394  INFO 119 --- [main] lavalink.server.config.KoeConfiguration  : Enabling JDA-NAS
2022-03-03 14:43:53.399  INFO 119 --- [main] c.s.l.c.natives.NativeLibraryLoader      : Native library udpqueue: loading with filter null
2022-03-03 14:43:53.404  INFO 119 --- [main] c.s.l.c.natives.NativeLibraryLoader      : Native library udpqueue: successfully loaded.
2022-03-03 14:43:53.411  INFO 119 --- [main] lavalink.server.config.KoeConfiguration  : Using Epoll Transport.
2022-03-03 14:43:53.412  INFO 119 --- [main] lavalink.server.config.KoeConfiguration  : Using the 'default' byte buf allocator
2022-03-03 14:43:53.422  INFO 119 --- [main] c.g.n.nativeloader.NativeLibLoader       : Native library timescale: loading with parts [Part{feature=X86.avx2, part=avx2}]
2022-03-03 14:43:53.438  INFO 119 --- [main] c.g.n.nativeloader.NativeLibLoader       : Native library detector: loading with parts []
2022-03-03 14:43:53.441  INFO 119 --- [main] c.g.n.nativeloader.NativeLibLoader       : Native library detector: successfully loaded.
2022-03-03 14:43:53.446  INFO 119 --- [main] c.g.n.nativeloader.NativeLibLoader       : Native library timescale: successfully loaded.
2022-03-03 14:43:53.447  INFO 119 --- [main] lavalink.server.io.SocketServer          : Timescale natives loaded successfully.
2022-03-03 14:43:54.093  INFO 119 --- [main] io.undertow                              : starting server: Undertow - 2.2.7.Final
2022-03-03 14:43:54.106  INFO 119 --- [main] org.xnio                                 : XNIO version 3.8.0.Final
2022-03-03 14:43:54.117  INFO 119 --- [main] org.xnio.nio                             : XNIO NIO Implementation Version 3.8.0.Final
2022-03-03 14:43:54.201  INFO 119 --- [main] org.jboss.threads                        : JBoss Threads version 3.1.0.Final
2022-03-03 14:43:54.262  INFO 119 --- [main] o.s.b.w.e.undertow.UndertowWebServer     : Undertow started on port(s) 443 (http)
2022-03-03 14:43:54.283  INFO 119 --- [main] lavalink.server.Launcher                 : Started Launcher in 5.359 seconds (JVM running for 6.288)
2022-03-03 14:43:54.285  INFO 119 --- [main] o.s.b.a.ApplicationAvailabilityBean      : Application availability state LivenessState changed to CORRECT
2022-03-03 14:43:54.287  INFO 119 --- [main] o.s.b.a.ApplicationAvailabilityBean      : Application availability state ReadinessState changed to ACCEPTING_TRAFFIC
2022-03-03 14:56:36.240  INFO 119 --- [XNIO-1 task-1] io.undertow.servlet                      : Initializing Spring DispatcherServlet 'dispatcherServlet'
2022-03-03 14:56:36.241  INFO 119 --- [XNIO-1 task-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2022-03-03 14:56:36.242  INFO 119 --- [XNIO-1 task-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 1 ms
2022-03-03 14:56:36.269 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:39896
2022-03-03 15:14:19.158 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:44840
2022-03-03 15:36:38.208 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:43754
2022-03-03 15:59:05.143 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:54808
2022-03-03 16:25:40.130 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:41088
2022-03-03 16:51:16.148 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:36274
2022-03-03 17:10:05.176 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:46758
2022-03-03 17:32:16.125 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:47418
2022-03-03 17:55:07.131 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:34678
2022-03-03 18:15:06.147 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:46102
2022-03-03 18:33:01.155 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:35192
2022-03-03 19:00:36.143 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:37984
2022-03-03 19:29:45.137 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:49386
2022-03-03 19:54:52.196 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:46788
2022-03-03 20:18:06.166 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:53860
2022-03-03 20:36:49.164 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:55266
2022-03-03 20:59:56.161 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:36456
2022-03-03 21:17:07.123 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:54112
2022-03-03 21:45:25.102 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:39076
2022-03-03 22:10:11.199 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:43260
2022-03-03 22:25:30.240 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:50930
2022-03-03 22:50:29.101 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:33224
2022-03-03 23:16:51.177 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:40554
2022-03-03 23:45:17.293 ERROR 119 --- [XNIO-1 task-1] l.server.io.HandshakeInterceptorImpl     : Authentication failed from /172.18.0.1:46448
